shared/chat/conversation/input-area/user-mention-hud/index.js

```diff
@@ -90,8 +90,8 @@ const hudStyle = {
   backgroundColor: globalColors.white,
 }
 
-const _withProps = ({users, filter, selectedIndex}) => ({
-  data: users
+const _withProps = ({users, filter, selectedIndex}) => {
+  const fullList = users
```

Comment:
hmm, i wonder where this list gets sorted

Reply:
at first glance, they're sorted in the `metaMap` so it may come already sorted